### PR TITLE
feat(cel-shed/p2p): bootstrappers connection tool

### PR DESCRIPTION
**Pull request by Wondertan**
Useful to debug connection issues with bootstrappers. It combined with go mod replaced libp2p I was able to quickly iterate over the code and find the root cause of the issue with had with our infrastructure

### CODE REVIEW

File: cmd/cel-shed/p2p.go [hunk 1]
```diff
@@ -75,3 +82,68 @@ var p2pPeerIDCmd = &cobra.Command{
...
+			fx.Provide(cmd.Context),
+			fx.Provide(store.Keystore),
+			fx.Provide(store.Datastore),
+			fx.Supply(bootstrappers),
+			fx.Supply(network),
+			fx.Supply(node.Light),
+			fx.Invoke(func(modprov p2p.Module) {
+				mod = modprov
+			}),
+		)
```
Comment (Wondertan) [inline cmd/cel-shed/p2p.go:112]: This shows we are far from the modular node

File: share/p2p/shrexsub/pb/notification.pb.go [hunk 2]
```diff
@@ -5,16 +5,19 @@ package share_p2p_shrex_sub
 
 import (
 	fmt "fmt"
-	proto "github.com/gogo/protobuf/proto"
 	io "io"
 	math "math"
 	math_bits "math/bits"
+
+	proto "github.com/gogo/protobuf/proto"
```
Comment (cristaloleg) [inline share/p2p/shrexsub/pb/notification.pb.go:12]: Something is not right with your local setup. AFAIK it should not format 🤔

File: nodebuilder/p2p/network.go [hunk 3]
```diff
@@ -69,6 +69,10 @@ var networkAliases = map[string]Network{
 	"private": Private,
 }
```
Comment (cristaloleg) [inline nodebuilder/p2p/network.go:71]: Doc?

File: cmd/cel-shed/p2p.go [hunk 4]
```diff
@@ -75,3 +82,68 @@ var p2pPeerIDCmd = &cobra.Command{
...
+				fmt.Printf("failed to stop application: %v\n", err)
+			}
+		}()
+
+		p2pInfo, err := mod.Info(cmd.Context())
+		if err != nil {
+			return err
+		}
+
+		fmt.Fprintf(os.Stdout, "PeerID: %s\n", p2pInfo.ID)
```
[the remark this answers is not in the document]
Comment (Wondertan) [reply cmd/cel-shed/p2p.go:130]: That's what copilot suggested and I didn't care about that at that moment

[the remark this answers is not in the document]
Comment (cristaloleg) [reply cmd/cel-shed/p2p.go:130]: Looks skewed and weird. On one line we specify the output on the other - not (`fmt.Println()`).

File: go.mod [hunk 5]
```diff
@@ -164,7 +164,7 @@ require (
 	github.com/golang/protobuf v1.5.4 // indirect
 	github.com/golang/snappy v0.0.5-0.20220116011046-fa5810519dcb // indirect
 	github.com/google/btree v1.1.2 // indirect
-	github.com/google/flatbuffers v1.12.1 // indirect
+	github.com/google/flatbuffers v2.0.8+incompatible // indirect
```
[the remark this answers is not in the document]
Comment (cristaloleg) [reply go.mod:167]: My `go mod tidy` doesn't change files. Consider to revert.

**Pull request closed without merging**